compiler-core/src/build/package_compiler.rs

```diff
@@ -219,6 +233,15 @@ where
                 "erl" => {
                     let _ = to_compile_modules.insert(relative_path.clone());
                 }
+                "ex" => {
+                    // Elixir core libs must be loaded
+                    maybe_link_elixir_libs(
```

Comment:
Why is it that we run this for every Elixir file? It looks like we could run it just once?